vault/dendron.release.2021-11-09.md

```diff
@@ -0,0 +1,76 @@
+---
+id: EjPEcAkZIDgzHaVMk4teA
+title: Release Notes (version 0.68)
+desc: ''
+updated: 1636480628276
+created: 1636478484110
+---
+
+Dendron 0.68 has sprouted  🌱
+
+Take a look at the latest [Dendron wiki](https://wiki.dendron.so), and you'll see some new features directly from these highlights:
+
+* Local TOC in the right-hand nav for each page in the published docs
+* Newly combined search bar that can go directly to a note, or search note contents using the `?` prefix
+
+Also: an enhancement in this release delivers breaking changes announced last week, due to some renaming of commands, which can be referenced in the changelog.
+
+### Highlights
+* feat(schemas): adding new command - create schema from hierarchy
+* feat(views): native dendron preview
+* feat(publish): add local table of contents, of current note, in right-hand nav
+* feat(publish): merge lookup and search fields
+* enhance(publish): migrate publish related commands to nextjs
+
+### Everything Else
+* enhance(schemas): adding error messaging and additional validation for when schemas are malformed
+* enhance(schemas): cmd create-schema-from-hierarchy allow schema to end with `*`
+* enhance(schema): update schemas on save instead of having to reload workspace
+* enhance(commands): move header supports multi vault
+* enhance(commands): allow creation of new notes when move header destination doesn't exist yet
+* fix(pods): github import pod handle deleted authors
+* fix(workspace): warning to remove stub property if the note has content
+* fix(publish): table of contents layout
+* fix(server): improving error response on api server
+* fix(workspace): Change Workspace command recognizes native workspaces
+
+### Community
+
+#### General Updates
+
+- New Discord channel, `#today-i-learned`, is a great place to share discoveries about Dendron and anything else!
+- Make sure to subscribe to the [weekly newsletter](https://buttondown.email/dendron), which kicked off last week
```

Comment:
maybe announce that its being revamped? 
eg. `We are making some changes to our newsletter (heads up, we have a newsletter). We are changing it from a monthly (lets be honest, once every few months) newsletter to a weekly one. The newsletter will feature {blah}. For more details, see {blah}} 

- the second blah should point to `community.email-newsletter` which is a note that describes what the newsletter is about